src/js/components/Heading/StyledHeading.js

```diff
@@ -10,11 +10,12 @@ const sizeStyle = props => {
   const levelStyle = headingTheme.level[props.level];
   if (levelStyle) {
     const data = levelStyle[size];
-    const styles = [
+    const styles =  
+    [
       css`
-        font-size: ${data.size};
-        line-height: ${data.height};
-        max-width: ${data.maxWidth};
+        font-size: ${data ? data.size : size};
+        line-height: ${data ? data.height : 'normal'};
+        max-width: ${data ? data.maxWidth : levelStyle.medium.maxWidth};
```

Comment:
Potential issue here:
If a custom theme has been defined and that theme doesn't have a levelStyle for medium, this will result in an error. 
Alternatives: Note set any maxWidth (so set to `unset`) or define a standard maxWidth to be used when medium is not available
Please advice :)